firebase connection for android and web

yunusemreshahbaz opened the pull request.

yunusemreshahbaz committed:
firebase connection for android and web

Yuaz3578-51 commented on android/app/build.gradle:
```diff
@@ -65,3 +66,4 @@ flutter {
 }
 
 dependencies {}
+// apply plugin: 'com.google.gms.google-services'
```
this should be deleted

yunusemreshahbaz committed:
Cards added to display homes

yunusemreshahbaz committed:
firebase connected

firebase is connected to the project, later changes should be applied asap for it to work as we planned
